SnapMD.VirtualCare.ApiModels/ScheduledConsultationResult.cs

```diff
@@ -13,5 +14,10 @@ public class ScheduledConsultationResult
         /// The ID of the newly-created consultation.
         /// </summary>
         public int ConsultationId { get; set; }
+
+        /// <summary>
+        /// The UID of the newly-created consultation request.
```

Comment:
@lorddev  The idea is to distinguish the GUID based unique ID from the INT based ID.